testing-modules/testing-assertions/src/test/java/com/baeldung/optional/OptionalUnitTest.java

```diff
@@ -0,0 +1,69 @@
+package com.baeldung.optional;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class OptionalUnitTest {
+
+    private final String stringToTest = "testString";
```

Comment:
This should probably be `private static final String EXPECTED_STRING` - in that there's no instance-level aspect to it.

Similarly, this isn't the string we're testing - it's the result we're expecting in our tests.

Reply:
ok i'll change it accordingly